packages/core/src/consumed-thing.ts

```diff
@@ -275,7 +275,7 @@ export default class ConsumedThing extends TD.Thing implements WoT.ConsumedThing
                 let input;
 
                 if (parameter !== undefined) {
-                    input = ContentManager.valueToContent(parameter, <any>this, form.contentType);
+                    input = ContentManager.valueToContent(parameter, <any>ta, form.contentType);
```

Comment:
Change should be reverted?

Reply:
Actually here I do not understand why the whole Consumed Thing is passed down to the ContentSerdes  instead of only the Action Affordance. In my case I would have some problems in the Content Serdes. Maybe can you explain me why it is like this only for actions?

Reply:
You are right, this was a mistake on our side. Thanks!

Reply:
ok, good!